pom.xml

```diff
@@ -4,7 +4,8 @@
     <modelVersion>4.0.0</modelVersion>
     <groupId>com.exasol</groupId>
     <artifactId>virtual-schema-common-document-files</artifactId>
-    <version>0.2.0-SNAPSHOT</version>
+    <version>0.2.0-
+    </version>
```

Comment:
Do you need to remove the `-` at the end?

Reply:
I changed it back to snapshot again.